buddyboss/buddyboss-platform
# PROD-762 #2793 - fix: Cannot translate the Profile Photo and Cover Photo on Elementor Profile Completion widget

### All Submissions:






### Changes proposed in this Pull Request:



Fixes https://github.com/buddyboss/buddyboss-platform/issues/2793 .

### How to test the changes in this Pull Request:

1. Create an Elementor page
2. Insert Profile Completion widget. https://i.ibb.co/RHsV6Nm/profile.jpg
3. Make sure that you enable Profile and Cover Photo.
4. Now, using the Loco Translate plugin, try to translate all the instances of **"Profile Photo"** and **"Cover Photo"**.
5. Go to BuddyBoss > Tools > Repair Community
6. Check/Enable **Repair default profile set and fields** option and click on **Repair Items** button
7. Lastly, go to frontend and notice those strings are now translated

### Proof Screenshots or Video
https://vimeo.com/588713860/551918f09c



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
